scripts/update-tools.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -ue
+dominikh="$(curl https://api.github.com/repos/dominikh/go-tools/releases/latest -s | jq -r .tag_name)"
+sed -i'' "s/staticcheck@.*$/staticcheck@$dominikh/" Dockerfile.jenkins
+
+golangci="$(curl https://api.github.com/repos/golangci/golangci-lint/releases -s | jq -r '.[].tag_name' | sort -rV | head -n 1)"
+sed -i'' "/golangci-lint.sh/s/v[0-9.]*$/$golangci/" Dockerfile.jenkins
+sed -i'' "/golangci-lint.sh/s/v[0-9.]*$/$golangci/" Makefile
+git diff
```

Comment:
I would prefer not to launch the pager with this command. It looked a bit confusing to me when I run the command and there were no changes, made me think, for 3 seconds, that my computer was frozen.

Could we use `git --no-pager diff`?

Reply:
Weird, this doesn't happen on my system. The diff is shown, then I'm immediately returned to a command prompt. On a fully updated repo, this does take ~38 seconds and produce no output.

I've started a PR for progress: https://github.com/storj/storj/pull/3572

Reply:
That's probably because I have the `$PAGER` env variable defined in my machine.

`--no-pager` will ensure that it outputs without it.